### PR TITLE
Euro tech test automation/t001/login and logout func

### PR DESCRIPTION
**Pull request by NurettinCerrah**
Hello guys,

I've created login and logout functions for my application. After a while, I had to make some changes to my code. Please review my code and give feedback if needed.

### CODE REVIEW

File: src/test/java/LoginFunc.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
 public class LoginFunc {
     public static void main(String[] args) {
         System.out.println("Login Page Created");
-        System.out.println("Username box created");
-        System.out.println("Password box created");
+        System.out.println("Username box update");
+        System.out.println("Password box updated");
         System.out.println("Login button created");
```
Comment (ATABEG00) [inline src/test/java/LoginFunc.java:6]: Maybe you can create "Forget Password?" as well

Comment (Gujalwar) [reply src/test/java/LoginFunc.java:6]: You can also think of creating a "Sign up" functionality.

File: src/test/java/LoginFunc.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-        System.out.println("After entering valid data redirection created");
+        System.out.println("Forget Password feature added");
```
Comment (NurettinCerrah) [inline src/test/java/LoginFunc.java:7]: @ATABEG00 the func added to line 7

**Review by ATABEG00 — COMMENTED** [review]
Perfect!

**Review by ATABEG00 — COMMENTED** [review]
Perfect

**Pull request merged**